flybywiresim/a32nx
# fix(mcdu): AOC RCVD MSGS scrolling, add arrows, and update on receipt

Fixes #2073

## Summary of Changes

- Fix scrolling of AOC RCVD MSGS pages per #2073.
- Add arrows to indicate which direction page can be scrolled.
- Update the page on receipt of a new message if it is active.

## Screenshots (if necessary)


Scroll available to the left:
![image](https://user-images.githubusercontent.com/9995998/106261356-553af900-6286-11eb-8d9c-b760aad7bf5d.png)


## References




Unaware of any references for this functionality, so excerised judgment.

## Additional context



Discord username (if different from GitHub): tracernz#3313

## Testing instructions

Send multiple ATIS and/or METAR requests, and navigate to received messages page.
- Observe the messages appearing in the list as they are received
- Check scrolling back and forth between pages always works, and cannot scroll past first/last pages
- Delete all messages on the last page (with multiple pages available) and ensure the display scrolls back to the new last page
- Correct messages are displayed or deleted (with CLR in scratchpad) when pressing LSK next to a message


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link
